apps/server/src/modules/user-import/uc/user-import.uc.spec.ts

```diff
@@ -353,5 +360,121 @@ describe('[ImportUserModule]', () => {
 				});
 			});
 		});
+
+		describe('[saveAllUsersMatches]', () => {
+			let system: System;
+			let school: School;
+			let currentUser: User;
+			let userMatch1: User;
+			let userMatch2: User;
+			let importUser1: ImportUser;
+			let importUser2: ImportUser;
+			beforeEach(() => {
+				system = systemFactory.buildWithId();
```

Comment:
Do we really need the id in all these entities? Mostly the entity reference should be enough even if we are checking equality of entities. See Mikro-ORM unit of work:
https://mikro-orm.io/docs/unit-of-work